incognia.go

```diff
@@ -323,7 +324,7 @@ func (c *Client) registerLogin(login *Login) (*TransactionAssessment, error) {
 		return nil, ErrMissingLogin
 	}
 
-	if login.InstallationID == "" {
+	if login.InstallationID == nil && login.SessionToken == nil {
 		return nil, ErrMissingInstallationID
 	}
```

Comment:
Shouldn't this be MissingInstallationIDOrSessionToken?

Reply:
Do u mean the error that is raised? U are correct, Should I raise different errors or combine them in only one?

Reply:
I believe it is not possible to raise different errors, as if both are null, it's not obvious which one should be present.
I see no problem in combining in a single error.